tests/test_replace_im_links.py

```diff
@@ -0,0 +1,361 @@
+import os
+from mbtools import replace_im_links
+from mbtools.fetch_im_resources import IM_PREFIX
+from mbtools import utils
+from lxml import html
+import json
+
+
+def collect_resources_from_mbz(mbz_path, prefix=None):
+    """
+    Given a path to an opened mbz file, this function will return all of the
+    references to external sources referenced by the HTML within that mbz. If a
+    prefix is provided, it will return the references that contain that
+    prefix
+    """
+    elems = []
+    elems.extend(utils.parse_backup_elements(mbz_path))
+    resource_urls = []
+    for elem in elems:
+        resource_urls.extend(utils.find_references_containing(
```

Comment:
Per my other related comment, I would not use / modify `find_references_containing` for the purposes of test code. Instead write this helper function used in your tests to be self-standing.